api/profile/profileRouter.js

```diff
@@ -218,7 +217,7 @@ router.post('/', authRequired, async (req, res) => {
  *                profile:
```

Comment:
let's be sure when we add code comments to add a bit more description about 'why' the comment is there.